### PR TITLE
fix(plugin): update critical attributes list in verify-signature

### PR DESCRIPTION
**Pull request by gokarnm**
Updated the missing critical attributes in `verify-signature` command.

Signed-off-by: Milind Gokarn <gokarnm@amazon.com>

### CODE REVIEW

**Review by rgnote — APPROVED** [review]
LGTM

File: specs/plugin-extensibility.md [hunk 1]
```diff
@@ -479,13 +479,20 @@ This allows Notary v2 implementations to perform the same logic themselves, if t
...
-       // One of notary.default.x509 or notary.signingAuthority.x509
-       "signingScheme" : "notary.default.x509",
+       // One of notary.x509 or notary.signingAuthority.x509
+       "signingScheme" : "notary.x509" | "notary.signingAuthority.x509",
        // Value is always RFC 3339 formatted date time string
-       "expiry": "2022-10-06T07:01:20Z", 
+       "expiry": "2022-10-06T07:01:20Z",
+       // Populate authenticSigningTime critical attibute 
+       // if signingScheme is notary.signingAuthority.x509.
+       "authenticSigningTime": "2022-04-06T07:01:20Z",
```
Comment (priteshbandi) [inline specs/plugin-extensibility.md]: Should we reword this to say
`authenticSigningTime must be present, if signingScheme is notary.signingAuthority.x509.`

**Review by priteshbandi — APPROVED** [review]
LGTM

File: specs/plugin-extensibility.md [hunk 2]
```diff
@@ -479,13 +479,20 @@ This allows Notary v2 implementations to perform the same logic themselves, if t
...
        // Value is always RFC 3339 formatted date time string
-       "expiry": "2022-10-06T07:01:20Z", 
+       "expiry": "2022-10-06T07:01:20Z",
+       // authenticSigningTime must be present, 
+       // if signingScheme is notary.signingAuthority.x509.
+       "authenticSigningTime": "2022-04-06T07:01:20Z",
+       // Name of the verification plugin
+       "verificationPlugin": "com.example.nv2plugin",
+       // Optional plugin minimum version, if present.
+       "verificationPluginMinVersion": "1.1.0.0",
```
Comment (shizhMSFT) [inline specs/plugin-extensibility.md:492]: The plugin version does not look like a SerVer2 version.

**Comment by gokarnm** [comment]
Resolving this PR, as these are addressed by #231 .
